vunit/test/unit/test_simulator_interface.py

```diff
@@ -53,6 +53,33 @@ def test_compile_source_files(self):
 """)
         self.assertEqual(project.get_files_in_compile_order(incremental=True), [])
 
+    def test_compile_source_files_minimal_subset(self):
```

Comment:
This is a lot of code just to test the if-statement in the SimulatorInterface class selecting the Project method to call. Maybe this test can be removed.